config.def.h

```diff
@@ -158,6 +158,7 @@ static const keymap_t keys[] = {
 	{ 0,            XK_J,             i_scroll_to_edge,     DIR_DOWN },
 	{ 0,            XK_K,             i_scroll_to_edge,     DIR_UP },
 	{ 0,            XK_L,             i_scroll_to_edge,     DIR_RIGHT },
+	{ 0,            XK_c,             i_scroll_to_center,   None },
```

Comment:
Is <kbd>z</kbd> taken? If not then I'd use <kbd>z</kbd>, since vim uses <kbd>zz</kbd> for centering.

Reply:
Besides basic movement and goto, the sxiv keybinds don't correspond closely to vi. I think `c` for center is a more obvious mnemonic.

Reply:
The other scroll bindings are using vi like bindings `H/J/K/L`, I don't think there's any reason to deviate for scroll to center.

Reply:
I'll refrain from bikeshedding. `z` is acceptable.